### PR TITLE
10181 add module elico_webkit_logo

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by elicoidal** [comment]
Change name and remove elico
Eric CAUDAL

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: webkit_logo_header/README.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* Jon chow <jon.chow@elico-corp.com>
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
```
Comment (elicoidal) [inline webkit_logo_header/README.rst]: Elico Maintainer

File: webkit_logo_header/data.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
...2vA7pLPysW7thVQsd0gK3nnax2lvQyWMovjp27H2MgEA94yNXSTl9qpkfgpD8w4A2VoRlitXUy3vpnqyz0A3pBNgIfUmKgu7KlRYR+40sjXEU6m6WlCRlp6zN09VQHbV2gMA9Op1aC02CqkyGq44Uq02r+vi6FV1N6e7bT5h1LKoAdeDV0wg4odjxRU8meJ4ehETT4JV9FRVc5n/L7ZdVCFsg8uK7tDB7Z71vSJ50RMPuU+uJ9VTAIq8Wik66BYeZKWRqD1WZZRQHo9YkV5RAiR13znT/kG/j6uNuqnuIdseV+9DodQ6t2rnM7EZlKEejTMW6jhRLai08q1wyq7aaGcT4euRoFjTP0aQp0C0jZwSgLx9yxxQJXxP6fpXf4KGvO+Jj3z604H+5B6wGp1zz8aGPxgjBICbO2N3HtbTk7cpOfZZhdPY4djeyF7q8VGhNkFVdknQCgqQ9Z4VApDoTaHV/pTMWihOOterDFISiLTygdTGOAHFvJF6cz1RvatPktkFiJTMmjMKArH1XynAUliB0F/9KIJxSReBc3ewk/M9wuHoUNRN2GIfvhH5Jk7W39yhTX9JBDxj878RADYgP/91WP1Nu9dgxElunBi7+aBWz6NtiBOIKHlWGWJ1tJaoHVXjAoIUy8WigDFZN9EyYgIDqTBej8jK/aorVXnWRBkYM9TelAmoZd+Zdakr+plUZAea0A9IAAZ7+ZG0yV/hqBI5Kw1DmhYFWZwEgmmRMC+L7WOOU8uoYes9Ozmim9PGMlayRhtkvgb5uadi1TftVzlGWHnEaaCfz8jS6XWxEsv76Gk3RPtrqfJEVCs82NUEWYWFKSKSetaZmOsQLKh+bfV52ttSpl+TWN+fKY9I1covqgB17RF71XuUwKd6b1JS5DSqGyS8IVF4IbqZvk+BqSESUN0FivH3I+IVyRg4R0M3B/9FDR5xDPKn12LbO96P9X+IncOcpvv+8p7B+Qt720vbSJekwCJBmAmi7GmErkNjFwKkW25mO32in8aCwEklUW2TXZXHImhiQvV648U8cqmuhgk61MNah9uqkRuRe4tuNNMGffo1kkCdWgSNIuGQvKlhsnbtkaV00Jy7hfmW6x0rGix73i1WMbtGSdG8kp2sthYjJrtSzktgU4ly2TDkjbdh9IpPYctDTXf5/wemPGjv1R2B7AAAAABJRU5ErkJggg==
```
Comment (elicoidal) [inline webkit_logo_header/data.xml:8]: What is this logo?

**Comment by siyuancn** [comment]
Please user the structure:
A module is organised in a few directory:

```
controllers/: contains controllers (http routes)
data/: data xml
demo/: demo xml
models/: models definition
report/: reporting models (BI/analysis), Webkit/RML print report templates
static/: contains the web assets, separated into css/, js/, img/, lib/, ...
views/: contains the views and templates, and QWeb report print templates
wizards/: wizard model and views
```

File: webkit_logo_header/data.xml (hunk 2 above)
Comment (Rona111) [inline webkit_logo_header/data.xml:8]: @elicoidal maybe this module needs double check.

Comment (elicoidal) [inline webkit_logo_header/data.xml:8]: Cancel this PR

**Comment by Rona111** [comment]
@elicoidal  need double check ,closed.

**Comment by noahzaozao** [comment]
close this PR , we've confirmed on Skype @elicoidal

**Comment by elicoidal** [comment]
Agree

**Pull request closed without merging**